packages/addons/addon-depends/pcsc-lite/package.mk

```diff
@@ -17,12 +17,12 @@
 ################################################################################
 
 PKG_NAME="pcsc-lite"
-PKG_VERSION="1.8.16"
-PKG_SHA256="e7d08aa38897e86fdf632d56ac70663a3a9add3c0bf4031dc32e783f19c0688a"
+PKG_VERSION="1.8.22"
+PKG_SHA256="6a358f61ed3b66a7f6e1f4e794a94c7be4c81b7a58ec360c33791e8d7d9bd405"
 PKG_ARCH="any"
 PKG_LICENSE="GPL"
 PKG_SITE="http://pcsclite.alioth.debian.org/pcsclite.html"
-PKG_URL="https://alioth.debian.org/frs/download.php/file/4164/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
+PKG_URL="https://alioth.debian.org/frs/download.php/file/${PKG_VER}/${PKG_NAME}-${PKG_VERSION}.tar.bz2"
```

Comment:
`PKG_VER` is no longer defined, `PKG_VER="4225"` was removed

Reply:
`PKG_URL="https://alioth.debian.org/frs/download.php/latestfile/39/${PKG_NAME}-${PKG_VERSION}.tar.bz2"` works for different versions

damn, read `PKG_VER` as `PKG_VERSION`, upps

Reply:
@CvH told me to remove it -_- should of noticed this myself

Rebased again with it added

Reply:
don't add PKG_VER again, just use `PKG_URL="https://alioth.debian.org/frs/download.php/latestfile/39/${PKG_NAME}-${PKG_VERSION}.tar.bz2"` , that works without that extra var :)

Reply:
Rebased again for the xxnth time, hope that does it now :)